Package.swift

```diff
@@ -24,6 +24,11 @@
 
 import PackageDescription
 
+// To use an local installation of ArcGIS Runtime for iOS, uncomment the following lines and also uncomment
+// package depenencies:package:path and comment the depenencies:package:url
+// import class Foundation.ProcessInfo
+// let sdk_install_package:String = (ProcessInfo.processInfo.environment["HOME"] ?? "") + "/Library/SDKs/ArcGIS"
```

Comment:
```suggestion
// let sdk_install_package: String = (ProcessInfo.processInfo.environment["HOME"] ?? "") + "/Library/SDKs/ArcGIS"
```
FYI last time Zach recommended using `import FileProvider` and `FileManager.default.homeDirectoryForCurrentUser.path`. I'll leave it to Zach's comments.